apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1294,6 +1300,14 @@ export class RushConfiguration {
     return this._gitChangeLogUpdateCommitMessage;
   }
 
+  /**
+   * [Part of the "gitPolicy" feature.]
+   * The separator between package name and version in git tag.
```

Comment:
The slash (`/`)  character seems to be the standard delimiter that Git uses for path components.  My Git client actually parses the `/` from our tags and displays a tree similar to with Git branch paths:

![Capture](https://user-images.githubusercontent.com/4673363/132265606-e69ed077-0599-45db-9459-95cc4ff601a4.PNG)

- Current syntax example: `@rushstack/stream-collator_v4.0.119`
- @ziofat's proposed syntax: `@rushstack/stream-collator/v4.0.119`

Maybe @ziofat's syntax is actually a superior approach and we should change it to be the default.  Then instead of `gitTagSeparator = "/"` maybe the setting should be `legacyGitTagSeparator = false`?

As far as allowing an arbitrary string -- would anyone realistically ever specific a string besides `/` or `_`?

Reply:
I could imagine `.` or `-`. Not saying I'd use it, but I could imagine it.

Reply:
`-` is used widely. An example would be [rollup plugins](https://github.com/rollup/plugins/tags).